doc/COMPILING/COMPILING-MSYS.md

```diff
@@ -1,48 +1,50 @@
+These instructions were written using 64-bit Windows 7 and the 64-bit version of MSYS2; the steps should be the same for other versions of Windows.
 
 ## Prerequisites:
 
-* Computer with 64 bit version of modern Windows operating system installed (Windows 10, Windows 8.1 or Windows 7);
-* NTFS partition with ~10 Gb free space (~2 Gb for MSYS2 installation, ~3 Gb for repository and ~5 Gb for ccache);
-* 64 bit version of MSYS2 (installer can be downloaded from [MSYS2 homepage](http://www.msys2.org/));
+* Windows 7, 8, 8.1, or 10
+* NTFS partition with ~10 Gb free space (~2 Gb for MSYS2 installation, ~3 Gb for repository and ~5 Gb for ccache)
+* 64-bit version of MSYS2
 
 **Note:** Windows XP is unsupported!
 
 ## Installation:
 
-1. Go to [MSYS2 homepage](http://www.msys2.org/) and download 64 bit installer (e.g. [msys2-x86_64-20180531.exe](http://repo.msys2.org/distrib/x86_64/msys2-x86_64-20180531.exe)).
+1. Go to the [MSYS2 homepage](http://www.msys2.org/) and download the installer.
 
-2. Run downloaded file and install MSYS2 (click `Next` button, specify directory where MSYS2 64 bit will be installed (e.g. `C:\msys64`), click `Next` button again, specify Start Menu folder name and click `Install` button).
+2. Run the installer. It is suggested that you install to a dev-specific folder (C:\dev\msys64\ or similar), but it's not strictly necessary.
 
-3. After MSYS2 installation is complete press `Next` button, tick `Run MSYS2 64 bit now` checkbox and press `Finish` button.
+3. After installation, run MSYS2 64bit now.
 
 ## Configuration:
 
-1. Update the package database and core system packages with:
+1. Update the package database and core system packages:
 
 ```bash
-pacman -Syu
+pacman -Syyu
 ```
 
-2. If asked close MSYS2 window and restart it from Start Menu or `C:\msys64\msys2_shell.cmd`.
+2. MSYS will inform you of a cygheap base mismatch and inform you a forked process died unexpectedly; these errors appear to be due to the nature of `pacman`'s upgrades and *may be safely ignored.* You will be prompted to close the terminal window; do so, then re-start using the MSYS2 MinGW 64-bit menu item.
 
-3. Update remaining packages with:
+3. Update remaining packages:
 
 ```bash
 pacman -Su
 ```
 
-4. Install packages required for compilation with:
+4. Install packages required for compilation:
 
 ```bash
-pacman -S git git-extras make mingw-w64-x86_64-{astyle,ccache,gcc,libmad,libwebp,ncurses,pkg-config,SDL2} mingw-w64-x86_64-SDL2_{image,mixer,ttf}
+pacman -S git git-extras make mingw-w64-x86_64-{astyle,ccache,gcc,libmad,libwebp,pkg-config,SDL2} mingw-w64-x86_64-SDL2_{image,mixer,ttf}
 ```
 
-5. Update paths in system-wide profile file (e.g. `C:\msys64\etc\profile`) as following:
+5. Close MSYS2.
+
+6. Update path variables in the system-wide profile file (e.g. `C:\msys64\etc\profile`) as following:
```

Comment:
Make sure to review and update all paths in whole document (you recommended `C:\dev\msys64` before).

Reply:
I swear I changed that. I must have lost the change somewhere along the line when playing musical files. I appreciate the catch.